dev-docs/bidders/exadsadserver.md

```diff
@@ -0,0 +1,484 @@
+title: EXADS
+description: EXADS Bidder Adapter
+pbjs: true
+biddercode: exadsadserver
+gvl_id: 1084
+tcfeu_supported: true
+media_types: banner, video, native
+gpp_sids: tcfeu
+safeframes_ok: false
+deals_supported: false
+floors_supported: true
+ortb_blocking_supported: true
+---
+
+### Example
+
+A very minimal RTB Banner example that shows how to use the EXADS adapter.
+The most important attributes are: **endpoint**, **fid** and **zoneId**. You can get them after configuring the zones.
+For more details about ad formats and parameters, read it in the next sections.
+
+```js
+
+adUnits = [
+  {
+    code: 'postbid_iframe', // the frame where to render the creative
+    mediaTypes: {
+      banner: {
+        sizes: [300, 250],
+      },
+    },
+    bids: [
+      {
+        bidder: 'exadsadserver',
+        params: {
+          zoneId: 12345,
+          fid: '829a896f011475d50da0d82cfdd1af8d9cdb07ff',
+          partner: 'ortb_2_4',
+          siteId: '123',
+          siteName: 'test.com',
+          country: 'IRL,
+          userIp: '0.0.0.0',
+          userId: '1234',
+          impressionId: impression_id.toString(),
+          mimes: ['image/jpg'],
+          endpoint: 'https://your-ad-network.com/rtb.php',
+        },
+      },
+    ],
+  },
+];
+```
+
+### Configuration
+
+#### General settings
+
+Use `setConfig` to instruct Prebid.js to initilize the exadsBidAdapter, as specified below.
+
+* Set "debug" as true if you need to read logs;
+* Set "gdprApplies" as true if you need to pass gdpr consent string;
+* The tcString is the iabtcf consent string for gdpr;
+* Uncomment the cache instruction if you need to configure a cache server (e.g. for instream video)
+
+```js
+pbjs.setConfig({
+  debug: false,
+  //cache: { url: "https://prebid.adnxs.com/pbc/v1/cache" },
+  consentManagement: {
+    gdpr: {
+      cmpApi: 'static',
+      timeout: 1000000,
+      defaultGdprScope: true,
+      consentData: {
+        getTCData: {
+          tcString: consentString,
+          gdprApplies: false, // set to true to pass the gdpr consent string
+        },
+      },
+    },
+  },
+});
+```
+
+##### Video settings
+
+If you will work with video stream ad formats you could choose to use the prebidJS video module to render the video using  already supported video players as videoJS.
+Add the `video` config if you need to render videos using the video module.
+For more info navigate to <https://docs.prebid.org/prebid-video/video-module.html>.
+
+```js
+pbjs.setConfig({
+  video: {
+    providers: [
+      {
+        divId: 'player', // the id related to the videojs tag in your body
+        vendorCode: 2, // videojs,
+        playerConfig: {
+          params: {
+            adPluginConfig: {
+              numRedirects: 10,
+            },
+            vendorConfig: {
+              controls: true,
+              autoplay: true,
+              preload: 'auto',
+            },
+          },
+        },
+      },
+    ],
+  },
+});
+```
```

Comment:
Is this specific to your adapter? Otherwise if this is already present in the prebid docs, I would remove it as well

Reply:
It is not specific to our adapter.
The idea was to give an immediate an immediate usage example to the end user.

Reply:
Removed this example code. It is not specific to our adapter.